instrumentation/active_storage/test/test_helper.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+ENV['RAILS_ENV'] = 'test'
+
```

Comment:
We recently had to add explicit requires to `'logger'` in the test helpers for some Rails-related libraries to avoid uninitialized constant errors: #1345 

Rails added `logger` as a dependency in Rails 7.1 and above, but since we still support 7.0, we need to require it here.
```suggestion

require 'logger'
```

Reply:
That makes sense, I added explicit `logger` require. https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/1313/commits/da97949a3c453e649105433c331b53e9c923c96c